content/docs/use-cases/kubeflow-on-multinode-cluster.md

```diff
@@ -0,0 +1,228 @@
+
+```shell
+kubectl get storageclass -n kubeflow
+
+NAME                   PROVISIONER                            AGE
+nfs (default)          cluster.local/nfs-client-provisioner   6h13m
+...
+```
+
+Notice that the installation command set the `storageClass.defaultClass` parameter to `true`. 
+This sets the *nfs* as the default storage class. 
+Therefore when you install Kubeflow, all PVCs will be labelled with the *nfs* storage class.
+
+## Finally: install Kubeflow!
+
+Now that you have prepared your on-prem, multi-node Kubernetes cluster to manage volumes using an NFS server, 
+you can install Kubeflow using the [Kubeflow on Kubernetes](/docs/started/getting-started-k8s/) guide.
+
+After installing Kubeflow, notice that that Kubernetes binds each PVC created by Kubeflow to an automatically created PV with the *nfs* storage class. Notice also that the provisioner has created a directory for each PVC inside the root NFS directory.
+
+## In case of existing Kubeflow installation
+
+If you set up Dynamic Provisioning *after* installing Kubeflow, you must change the storage class on your existing unbinded PVCs.
+
+To perform this task you need to:
+
+1. Download existing PVCs.
+2. Change their storage class.
+3. Delete and recreate them in the cluster.
+
+Download the three PVCs:
+
+```shell
+kubectl get pvc/mysql-pv-claim -n kubeflow -o yaml > mysql-pv-claim.yaml
+kubectl get pvc/minio-pvc -n kubeflow -o yaml > minio-pvc.yaml
+kubectl get pvc/katib-mysql -n kubeflow -o yaml > katib.yaml
+```
+
+And then modify files to add the right `storageClassName` under the `spec` section:
+
+```yaml
+# mysql-pv-claim.yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mysql-pv-claim
+  namespace: kubeflow
+  ...
+spec:
+  storageClassName: nfs
+  ...
+```
+
+```yaml
+# minio-pvc.yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: minio-pvc
+  namespace: kubeflow
+  ...
+spec:
+  storageClassName: nfs
+  ...
+```
+
+```yaml
+# katib.yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: katib-mysql
+  namespace: kubeflow
+  ...
+spec:
+  storageClassName: nfs
+  ...
+```
+
+Next, remove the old PVCs:
+
+```shell
+kubectl delete -f mysql-pv-claim.yaml
+kubectl delete -f minio-pvc.yaml
+kubectl delete -f katib.yaml
+```
+
+Finally, add the modified PVCs:
+
+```shell
+kubectl apply -f mysql-pv-claim.yaml
+kubectl apply -f minio-pvc.yaml
+kubectl apply -f katib.yaml
+```
+
+The PVCs are now bound to your NFS storage.
+
+## Limitations
+
+NFS is a remote filesystem that is high performant in reading but slower in writing.
+If you have to write a huge amount of data in your workflow, NFS might not be the right choice.
+
+Ensure that you are using version 4 of NFS (instead of version 3): NFS 3 can have some problems like 
+partial writing of documents and it does not support authentication. Ensure that you are using version 4 of NFS.
```

Comment:
Remove the last sentence (`Ensure that you are using version 4 of NFS.`) because you've already included this in the previous part :)